dogowner/migrations/0001_initial.py

```diff
@@ -0,0 +1,46 @@
+# Generated by Django 4.0.3 on 2022-03-23 01:21
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+import django.db.models.manager
+
+
+class Migration(migrations.Migration):
+    initial = True
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Dogowner',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True,
+                                           primary_key=True,
+                                           serialize=False,
+                                           verbose_name='ID')),
+                ('first_name', models.CharField(max_length=255)),
+                ('last_name', models.CharField(max_length=255)),
+                ('phone_number', models.CharField(max_length=20)),
```

Comment:
I think that max_length of phone_number should be 10, because there's no phone number that long, and to prevent typos.